integration-test-groups/jdbc/mysql/src/main/resources/application.properties

```diff
@@ -21,5 +21,6 @@
 
 quarkus.datasource.mysql.db-kind=mysql
 quarkus.datasource.mysql.jdbc.max-size=8
+quarkus.datasource.devservices.enabled=false
```

Comment:
Does this work when the `jdbc-grouped` itest module is run? All of the 'grouped' module `application.properties` will get combined, so this would effectively disable dev services for all of those JDBC tests.